docs/extensions.md

```diff
@@ -50,10 +50,10 @@ This ensures that no matter who is generating an XDM document, or how the XDM do
   "https://ns.example.com/asset_name": "custom_asset_1",
   "repo:id": "urn:aaid:a:b:01234578-0123-ABCD-abcd-0123456789ab",
   "xmp:createDate": "2017-09-26T15:52:25+00:00",
-  "repo:createdDate": "2017-09-26T15:52:25+00:00",
+  "repo:createDate": "2017-09-26T15:52:25+00:00",
```

Comment:
Yes, but I'd like to handle that in a separate PR. I only changed createdDate to createDate because it was already inconsistently defined in the schemas I was touching.

Reply:
Ok. Works for me.

Reply:
also remember @fmeschbe that we are also bound by existing terms from XMP (etc.) - which is why it's createDate and modifyDate.